apps/web/src/components/layout/components/v2/BridgeUpdateModal.tsx

```diff
@@ -124,7 +123,8 @@ export const BridgeUpdateModal: FC<BridgeUpdateModalProps> = ({ isOpen, toggleOp
                 <LinkText
                   onClick={(e) => {
                     e.preventDefault();
-                    window.open(`${DOCS_URL}/concepts/endpoint`, '_blank');
+                    setPath('concepts/endpoint');
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** We should centralize all paths like this in one file and refer to them by key. This will make it easier to reuse or modify them, and will enable us to use more type-safe params (than a string)‏